### PR TITLE
[integration_test] Fix channel name for iOS, docs

### PR DESCRIPTION
**Pull request by jiahaog**
## Description

Fix channel name for iOS, docs

I haven't published `0.8.1` yet, will do so after this PR to get the fix for the channel name in.

### CODE REVIEW

File: packages/integration_test/README.md [hunk 1]
```diff
@@ -47,28 +48,27 @@ import 'dart:async';
...
 
 Future<void> main() async => integrationDriver();
-
 ```
 
 To run a example app test with Flutter driver:
 
-```
+```sh
 cd example
```
Comment (jiahaog) [inline packages/integration_test/README.md:56]: I'm not really sure what this means... Is it referring to the example app in `packages/integration_test/example`? If so we should update the file names accordingly, and perhaps we should move the following docs there and direct users there from here.

Comment (dnfield) [reply packages/integration_test/README.md:56]: Yes, it's that directory.

File: packages/integration_test/ios/Classes/IntegrationTestPlugin.m [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #import "IntegrationTestPlugin.h"
 
-static NSString *const kIntegrationTestPluginChannel = @"plugins.flutter.io/integratoin_test";
```
Comment (dnfield) [inline packages/integration_test/ios/Classes/IntegrationTestPlugin.m:3]: I could have sworn @bparrishMines fixed this previously.

File: packages/integration_test/README.md (hunk 1 above)
Comment (jiahaog) [reply packages/integration_test/README.md:56]: Filed #64690, will follow up separately

File: packages/integration_test/ios/Classes/IntegrationTestPlugin.m (hunk 2 above)
Comment (bparrishMines) [reply packages/integration_test/ios/Classes/IntegrationTestPlugin.m:3]: Sorry, I was supposed to. I got distracted while waiting for our CI to go green.

**Pull request merged**